presto-sqlserver/src/test/java/io/prestosql/plugin/sqlserver/TestingSqlServerServer.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.sqlserver;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import io.prestosql.testing.TemporaryDockerContainer;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.Statement;
+
+import static java.lang.String.format;
+
+public final class TestingSqlServerServer
+{
+    private TestingSqlServerServer() {}
+
+    public static final String USER = "sa";
+    public static final String PASSWORD = "SQLServerPass1";
+
+    private static TemporaryDockerContainer dockerContainer = new TemporaryDockerContainer(
```

Comment:
As a general rule, you should not acquire external resources in static initializer.
This is especially applicable to external docker containers.

IMO, the best solution would be to run separate container per test class.
I think the container is pretty fast to start.